src/calculateRentalCost.js

```diff
@@ -4,7 +4,17 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sum = days * 40;
+
+  if (days >= 7) {
+    return (sum -= 50);
+  }
+
+  if (days >= 3) {
+    return (sum -= 20);
```

Comment:
As with the previous compound assignment, return the result of the expression directly instead of using `-=`.